internal/driver/driver.go

```diff
@@ -208,6 +211,25 @@ func applyCommandOverrides(cmd string, outputFormat int, cfg config) config {
 	return cfg
 }
 
+// withoutEmptyStrings filters a slice to only non-empty strings
+func withoutEmptyStrings(in []string) (out []string) {
+	for _, s := range in {
+		if s != "" {
+			out = append(out, s)
+		}
+	}
+	return
+}
+
+// generateLabelPseudoNodes generates extra nodes from the tagroot and tagleaf options.
+func generateLabelPseudoNodes(prof *profile.Profile, cfg config, ui plugin.UI) {
```

Comment:
Not a strong feeling but I'd name this generateTagRootsLeaves to bear resemblance to the flag names.

Reply:
I like your name better, changing.